recipes/orfanage/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.4_9f713d2" %}
+{% set commit_id = "9f713d2a55065bb952c29eee1272ad9a9ca09ee2" %}
+
+package:
+  name: "orfanage"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/alevar/ORFanage.git
```

Comment:
can you create a tarball release and then use that one?

Reply:
Thank you for reviewing my PR. After adding a tarball release  inplace of git_url I get the following error during Azure tests. Am I doing something wrong?

Error was:
Command '['/opt/conda/bin/git', 'clone', '--mirror', 'https://github.com/alevar/ORFanage/archive/refs/tags/1.0.4.tar.gz', '/opt/conda/conda-bld/git_cache/github.com/alevar/ORFanage/archive/refs/tags/1.0.4.tar.gz']' returned non-zero exit status 128.

Thank you,

Reply:
You can not clone a tarball, you need to use `url:`
Please read more in our documentation at: https://bioconda.github.io/contributor/index.html

Reply:
Thank you. I have updated the source and meta to comply with request.